src/js/main.js

```diff
@@ -37,6 +37,45 @@ class Site {
       $(this).html(string);
     });
   }
+
+  bindHoneypot() {
+    const $honeypot = $('#honeypot');
+
+    $honeypot.on('click', this.handleHoneypot.bind(this));
+  }
+
+  handleHoneypot() {
+    let honeypotCount = 0;
+    const honeypotInterval = setInterval(() => {
+      // oh well hello
+      const $honeypotImage = $('<img src="' + WP.themeUrl + '/dist/img/honeypot.jpg" class="honeypot" />');
+
+      // oooh that's sweet
+      $('body').append($honeypotImage);
```

Comment:
I think these appended images need to be removed as if you click it a few times you can end up with hundreds of `opacity: 0` images in your DOM lol.